packages/terra-table/CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ Changelog
 
 Unreleased
 ----------
+### Changed
+* Updated to allow Voice Over to read selectable table rows.
```

Comment:
nit: I believe [VoiceOver is typically concatenated](https://en.wikipedia.org/wiki/VoiceOver) when in written form.

Reply:
https://github.com/cerner/terra-core/pull/2331/commits/a7015c82ddf086876ec90d58d94a8a8aaeb9581d